src/lib/search-url-builder.ts

```diff
@@ -72,10 +72,12 @@ export const createUrl = (searchState: {
   query?: any
   refinementList?: any
   page?: number
+  sortBy?: string
```

Comment:
these changes make this "just work" as a query param and let's us persist state via the url